js/qmwp-mood-tracker.js

```diff
@@ -52,7 +53,8 @@ var onMoodButtonClicked = function () {
             url: api_host + "/api/measurements/v2",
             contentType: "application/json",
             headers: {
-                "Authorization": "Bearer " + access_token
+                "Authorization": "Bearer " + access_token,
```

Comment:
Identifier 'access_token' is not in camel case.<br>'access_token' is not defined.